meson.build

```diff
@@ -20,13 +20,16 @@ executable(
     'src/Application.vala',
     'src/ListView.vala',
     'src/MainWindow.vala',
+    'src/Services/Calendar/TaskStore.vala',
+    'src/Services/Calendar/Util/DateTime.vala',
+    'src/Services/Calendar/Util/ECalComponent.vala',
+    'src/Services/Calendar/Util/ICalTime.vala',
```

Comment:
This folder structure seems a little excessively nested

There's also currently an existing "Util.vala". All the Util classes should probably be in the same folder and I guess this needs to be renamed to be clear how it is different from these other Util classes